runbotics/runbotics-desktop/src/action/sap/sap.action-handler.ts

```diff
@@ -91,35 +132,81 @@ export default class SapActionHandler extends StatefulActionHandler {
         return array;
     }
 
+    /**
+     *  @alias focus 
+     *  @name SAP: Focus
+     *  @description Selects an element and "highlights" it.
+     *  @param target
+     *  @example focus('wnd[0]/example/example')
+     */
     async focus(input: SapTypes.SAPFocusActionInput): Promise<SapTypes.SAPFocusActionOutput> {
         this.isApplicationOpen();
         this.session.FindById(input.target).setFocus();
         return {};
     }
 
+    /**
+     *  @alias disconnect 
+     *  @name SAP: Disconnect
+     *  @description Disconnects from instance on which the bot is currently logged in.
+     */
     async disconnect() {
         await this.session?.Parent.CloseConnection();
         this.session = null;
     }
 
+    /**
+     *  @alias sendVKey 
+     *  @name SAP: Send Virtual Key
+     *  @description Indicates keyboard shortcut to be clicked by bot.
+     *  @param virtualKey
+     *  @example sendVKey(Enter)
+     *  @see SendVKeyMapper - check out more examples
+     */
     async sendVKey(input: SapTypes.SAPSendVKeyActionInput): Promise<SapTypes.SAPSendVKeyActionOutput> {
         this.isApplicationOpen();
         this.session.FindById('wnd[0]').SendVKey(SendVKeyMapper[input.virtualKey]);
         return {};
     }
 
+    /**
+     *  @alias readText
+     *  @name SAP: Read text
+     *  @description Reads text from indicated element.
+     *  @param target
+     *  @example readText(wnd[0]/example)
+     *  @returns text value from chosen element
+     */
     async readText(input: SapTypes.SAPReadTextActionInput): Promise<SapTypes.SAPReadTextActionOutput> {
         this.isApplicationOpen();
         const result = this.session.FindById(input.target).text;
         return result ? result.__value : null;
     }
 
+    /**
+     *  @alias select
+     *  @name SAP: Select
+     *  @description In some SAP transactions, 'Select' action is utilized to navigate to another tab. 
+     *  Additionally, 'Select' is frequently used to choose a particular option from the menu bar.
+     *  @param target
+     *  @example select(wnd[0]/mbar/menu[0]/menu[1])
+     */
     async select(input: SapTypes.SAPClickActionInput): Promise<SapTypes.SAPClickActionOutput> {
         this.isApplicationOpen();
         this.session.FindById(input.target).select();
         return {};
     }
 
+    /**
+     *  @alias openContextMenu 
+     *  @name SAP: Open Context Menu
+     *  @description In some SAP transactions, e.g. SE16N, in the LFB1 table view, there are toolbar icons with an arrow mark. 
```

Comment:
don't use `In some SAP transaction`, rephrase it

Reply:
dont put real life transactions, these are unique

Reply:
> dont put real life transactions, these are unique

`openContextMenu` uses elements(icons) that are not commonly available, for this reason I will leave the name of transaction and table, because these are universal in SAP.